packages/nuxt/README.md

```diff
@@ -140,6 +126,18 @@ Add an import flag to the node options, so the file loads before any other impor
 }
 ```
 
+If you are getting an `import-in-the-middle` error message, add the package with a minimum version of `1.10.0` as a
```

Comment:
l: I'm curious - Is there a concrete reason in Nuxt apps why users would run into this error? I think this is already rather specific for the SDK readme as it sounds a bit more like troubleshooting.

Generally, I'd tend towards keeping the Readmes in the repo rather short in favour of more detailed docs. They almost inevitably diverge at some point 🥲 But given we don't have docs yet for nuxt we can also leave this in and rethink if we need it once we give the readme a pass before going stable.

Reply:
The problem will most likely occur because of iitm-related issues with `vue/server-renderer` (repro [here](https://github.com/getsentry/sentry-javascript-examples/pull/38) and other issue comment [here](https://github.com/getsentry/sentry-javascript/issues/12114#issuecomment-2120096287)). That's why I added it here.

Reply:
we can look at this in a follow up, but maybe we can auto-add this package to the exclude config for the users 🤔